api/database/seeders/UserRandomSeeder.php

```diff
@@ -136,26 +133,6 @@ public function run()
                 }
             })
             ->create();
-
-        // attach either a work or education experience to a pool candidate to meet minimum criteria
-        PoolCandidate::all()->load('user')->each(function ($poolCandidate) {
-            $educationRequirementOption = $poolCandidate->education_requirement_option;
-            $user = $poolCandidate->user;
-
-            if ($educationRequirementOption === EducationRequirementOption::EDUCATION->name) {
-                //Ensure user has at least one education experience
-                $experience = EducationExperience::factory()->create([
```

Comment:
Repeats the logic in 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/739bad11-071c-4b83-b7e5-afe6ba533de6)